apps/docs/content/components/Bento.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: Bento
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/Bento/Bento.tsx
+storybook: '/brand/storybook/?path=/story/components-bento--default'
+description: Use the bento component to create a presentational layout for brand content
+---
+
+```js
+import {Bento} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Bento>
+  <Bento.Item columnSpan={3} rowSpan={4} />
+  <Bento.Item columnSpan={3} rowSpan={2} />
+  <Bento.Item columnSpan={6} rowSpan={4} />
+  <Bento.Item columnSpan={3} rowSpan={2} rowStart={5} />
+  <Bento.Item columnSpan={3} rowSpan={4} />
+  <Bento.Item columnSpan={6} rowSpan={2} />
+</Bento>
```

Comment:
Should the default example just be a single bento item? Or is this particular layout the standard one that we want folks to apply everywhere? 

![Screenshot 2023-08-31 at 09 17 32](https://github.com/primer/brand/assets/13340707/54c363d1-b02a-4bd2-84ce-8cb1bd90b71e)

Related, should Bento.Item span the entire width of the parent container by default, like we do with Grid.Column? Currently doesn't do this, but I think it should 🤔👇 


![Screenshot 2023-08-31 at 09 21 50](https://github.com/primer/brand/assets/13340707/8d556612-285b-417f-a323-a4e9430fa661)

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/390/commits/5036e104cc735ec7a6348df403eafc8beaa6416d)